src/css/components/files/_c-search-results.scss

```diff
@@ -0,0 +1,11 @@
+.c-search-results {
+  @include horizontal_align();
+  @include border_style(0.1em, 0.001em, black);
+  @include font(28px); 
+  background: rgba(255, 255, 255, 1.0);
+  padding: 6px;
+  box-sizing: border-box;
```

Comment:
These four rules `box-sizing`, `width`, `display`, `overflow-y` are used too many times with the same values. Avoid this by either using mixins, extend or utility rules.